front/components/data_source/DataSourceSyncChip.tsx

```diff
@@ -74,6 +75,25 @@ export default function ConnectorSyncingChip({
             trigger={<Chip color="warning">Synchronization failed</Chip>}
           />
         );
+      case "webcrawling_error_empty_content":
+        return (
+          <Tooltip
+            label={
+              "This problem commonly occurs with JavaScript-based websites," +
+              " as our current crawler cannot process JavaScript."
+            }
+            trigger={<Chip color="warning">Failed: no content</Chip>}
+          />
+        );
+      case "webcrawling_error_blocked":
+        return (
+          <Tooltip
+            label={
+              "This website has an added security layer that blocks crawlers."
```

Comment:
here would check copy with e.g. @gabhubert if he's got time

Reply:
This website's security settings prevent Dust from accessing it.